### PR TITLE
PAYARA-3539 Small cleanup cluster package - sweep 2

### PR DESCRIPTION
**Pull request by arjantijms**

### CODE REVIEW

**Comment by arjantijms** [comment]
Jenkins test please

File: nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/CreateNodeConfigCommand.java [hunk 1]
```diff
@@ -37,94 +37,118 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
-// Portions Copyright [2018] Payara Foundation and/or affiliates
+// Portions Copyright [2018-2019] Payara Foundation and/or affiliates
 
 package com.sun.enterprise.v3.admin.cluster;
 
-import com.sun.enterprise.config.serverbeans.Nodes;
+import static com.sun.enterprise.util.net.NetUtils.isThisHostLocal;
```
Comment (Cousjava) [inline nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/CreateNodeConfigCommand.java]: No static imports of functions

File: nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/CreateNodeSshCommand.java [hunk 2]
```diff
@@ -37,24 +37,35 @@
...
 package com.sun.enterprise.v3.admin.cluster;
 
-import com.sun.enterprise.config.serverbeans.Nodes;
-import java.util.List;
+import static com.sun.enterprise.v3.admin.cluster.NodeUtils.NODE_DEFAULT_REMOTE_USER;
+import static com.sun.enterprise.v3.admin.cluster.NodeUtils.NODE_DEFAULT_SSH_PORT;
+import static com.sun.enterprise.v3.admin.cluster.NodeUtils.PARAM_SSHKEYFILE;
+import static com.sun.enterprise.v3.admin.cluster.NodeUtils.PARAM_SSHKEYPASSPHRASE;
+import static org.glassfish.api.admin.RestEndpoint.OpType.POST;
+import static org.glassfish.cluster.ssh.util.SSHUtil.getExistingKeyFile;
```
Comment (Cousjava) [inline nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/CreateNodeSshCommand.java]: Don't import functions statically

File: nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/CreateNodeConfigCommand.java (hunk 1 above)
Comment (arjantijms) [reply nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/CreateNodeConfigCommand.java]: ???

**Pull request merged**